README_zh_CN.md

```diff
@@ -4,32 +4,38 @@ MinIO 是一个基于Apache License v2.0开源协议的对象存储服务。它
 
 MinIO是一个非常轻量的服务,可以很简单的和其他应用的结合，类似 NodeJS, Redis 或者 MySQL。
 
+[English Doc](README.md)
```

Comment:
no need to reference it here we reference it here https://docs.minio.io/

Reply:
ok,already updated.